prac_07/project_management.py

```diff
@@ -0,0 +1,86 @@
+"""
+Ashton Jack Stewart
+Project data saving and modification
+"""
+import datetime
+
+
+file_name = "projects.txt"
+
+menu = ("- (L)oad projects  \n"
+        "- (S)ave projects)  \n"
+        "- (D)isplay projects  \n"
+        "- (F)ilter projects by date \n"
+        "- (A)dd new project  \n"
+        "- (U)pdate project \n"
+        "- (Q)uit")
+
+#first_line_of_file = "Name	Start Date	Priority	Cost Estimate	Completion Percentage"
+first_line_of_file = ["Name",	"Start Date	Priority",	"Cost Estimate",	"Completion Percentage"]
+def main():
+
+    all_projects = []
+    chosen_action = "Not nothing"
+    while chosen_action != "Q":
+        print(menu)
+        chosen_action = input(">> ").upper()
```

Comment:
This is confusing, review patterns page.
[While loop](https://github.com/CP1404/Starter/wiki/Programming-Patterns#while-loops-indefinite-iteration)